packages/bot-web-ui/src/components/dashboard/joyride-config.tsx

```diff
@@ -94,6 +96,12 @@ const joyride_props: TJoyrideConfig = {
     showProgress: false,
     spotlightClicks: true,
     disableBeacon: true,
+};
+
+const onboard_tour_overlay: TTourOverlay = {
+    disableOverlay: false,
+};
+const bot_tour_overlay: TTourOverlay = {
     disableOverlay: true,
 };
 
```

Comment:
@rupato-deriv instead of creating 2 new objects just to pass one value I don't think it worth it.
could you please directly pass `disableOverlay: true` or `disableOverlay: false` to each of the steps?

Reply:
done ✅